### PR TITLE
add scaling option to the Spectrogram class and MelSpectrogram class

### PR DESCRIPTION
**Pull request by zoharrpg**
Expose mode option from scipy.  My implementation can pass the test, but I am not sure whether my code satisfy your expectation

### CODE REVIEW

**Comment by zoharrpg** [comment]
I've changed the mode option to a scaling option and submitted it to this branch. By default, the scaling will be "Spectrum", contrary to the documentation's default "density"
Thank you!

File: opensoundscape/spectrogram.py [hunk 1]
```diff
@@ -31,6 +31,10 @@ class Spectrogram:
             [default: none]
         audio_sample_rate: sample rate of audio from which spec was created
             [default: none]
+        scaling: { ‘density’, ‘spectrum’ },optional,
+        Selects between computing the power spectral density (‘density’) where Sxx has units of V**2/Hz and computing the power spectrum (‘spectrum’) where Sxx has units of V**2, if x is measured in V and fs is measured in Hz. Defaults to ‘density’.
+            [default: spectrum]
```
Comment (sammlapp) [inline opensoundscape/spectrogram.py:36]: Contradictory info - which is the default?

**Review by sammlapp — COMMENTED** [review]
The code looks good, but one docstring contains contradictory info about the Default value

**Comment by zoharrpg** [comment]
remove the wrong docs information

**Review by sammlapp — APPROVED** [review]

**Comment by zoharrpg** [comment]
I run and passed  the formatter and pytest on my local machine

**Pull request merged**